File: add/codesnippet/VisualBasic/t-system.web.configurati_64_1.vb
        ' Get the applicaqtion configuration.
        Dim configuration _
        As Configuration = _
        WebConfigurationManager.OpenWebConfiguration( _
        "/aspnetTest")

        ' Get the section.
        Dim anonymousIdentificationSection _
        As AnonymousIdentificationSection = _
        CType(configuration.GetSection( _
        "system.web/anonymousIdentification"), _
        AnonymousIdentificationSection)

File: add/codesnippet/CSharp/t-system.web.configurati_64_1.cs
            // Get the applicaqtion configuration.
            Configuration configuration =
                WebConfigurationManager.OpenWebConfiguration(
               "/aspnetTest");

            // Get the section.
            AnonymousIdentificationSection anonymousIdentificationSection = 
                (AnonymousIdentificationSection)configuration.GetSection(
                "system.web/anonymousIdentification");